src/tests/ftest/pool/pool_svc.yaml

```
@@ -7,6 +7,10 @@ hosts:
     - server-E
 server_config:
     name: daos_server
+    servers:
+        log_mask: DEBUG,MEM=ERR
+        env_vars:
+            - DD_MASK=mgmt,md,dsms,any
```

Specifying a value for `env_vars` replaces the default and the following values - which are normally set - will not be specified:
```
                "ABT_ENV_MAX_NUM_XSTREAMS=100",
                "ABT_MAX_NUM_XSTREAMS=100",
                "DAOS_MD_CAP=1024",
                "DD_MASK=mgmt,io,md,epc,rebuild",
                "D_LOG_FILE_APPEND_PID=1",
                "COVFILE=/tmp/test.cov"
```
Just in case this has an impact, I'd suggest the following edit:
```suggestion
        env_vars:
            - ABT_ENV_MAX_NUM_XSTREAMS=100
            - ABT_MAX_NUM_XSTREAMS=100
            - DAOS_MD_CAP=1024
            - DD_MASK=mgmt,md,dsms,any
            - D_LOG_FILE_APPEND_PID=1
            - COVFILE=/tmp/test.cov
```